File: app/src/main/java/com/mason/kakao/masonsgallary/util/TagUtil.java
package com.mason.kakao.masonsgallary.util;

import com.mason.kakao.masonsgallary.R;
import com.mason.kakao.masonsgallary.model.data.Tag;

/**
 * Created by kakao on 2017. 11. 2..
 */

public class TagUtil {
    public static int getResourceIdByTag(Tag tag) {
        int resourceId = 0;
        switch(tag) {
            case Ryan:
                resourceId = R.drawable.tag_ryan;
                break;
            case Muzi:
                resourceId = R.drawable.tag_muzi;
                break;
            case Apeach:
                resourceId = R.drawable.tag_apeach;
                break;
            case Frodo:
                resourceId = R.drawable.tag_frodo;
                break;
            case Neo:
                resourceId = R.drawable.tag_neo;
                break;
            case Tube:
                resourceId = R.drawable.tag_tube;
                break;
            case Jay_G:
                resourceId = R.drawable.tag_jay_g;
                break;
            case Con:
                resourceId = R.drawable.tag_con;
                break;
        }
        return resourceId;
    }
}

File: app/src/main/java/com/mason/kakao/masonsgallary/util/TagUtil.kt
package com.mason.kakao.masonsgallary.util

import com.mason.kakao.masonsgallary.R
import com.mason.kakao.masonsgallary.model.data.Tag

/**
 * Created by kakao on 2017. 10. 31..
 */

class TagUtil {
    companion object {
        fun getTagById(id: Int): Tag {
            when (id) {
                R.id.menu_ryan -> return Tag.Ryan
                R.id.menu_apeach -> return Tag.Apeach
                R.id.menu_muzi -> return Tag.Muzi
                R.id.menu_frodo -> return Tag.Frodo
                R.id.menu_neo -> return Tag.Neo
                R.id.menu_tube -> return Tag.Tube
                R.id.menu_jay_g -> return Tag.Jay_G
                R.id.menu_con -> return Tag.Con
            }
            return Tag.All
        }
    }
}
